### PR TITLE
Initial language for adding hourly support pricing

### PR DESCRIPTION
**Pull request by apburnes**
Looking for as much feedback as possible.

Proposed changes in this pull request:
- Add support package to pricing page
- Proposing 100 hours 
- An list of the package's deliverables

[CircleCI](https://circleci.com/gh/18F/federalist.18f.gov/tree/apb%2Fsupport-pricing)

[:sunglasses: PREVIEW](https://federalist-a44a7672-afb1-473e-acb2-067f644c0967.app.cloud.gov/preview/18f/federalist.18f.gov/apb/support-pricing/pricing/)

cc @RonWilliams

### CODE REVIEW

File: _data/pricing.yml [hunk 1]
```diff
@@ -9,3 +9,9 @@ plans:
       - { text: Automatic HTTPS certificates and renewals for all your websites. }
       - { text: Unlimited sites for your office. }
       - { text: No charges for storage or bandwidth. }
+  - title: "Platform Support $27,500/100 hours"
+    features:
+      - { text: "Personalized on-boarding with review of Federalist features and Github integrations." }
```
Comment (davemcorwin) [inline _data/pricing.yml:14]:
```suggestion
      - { text: "Personalized on-boarding with review of Federalist features and Github integration." }
```

File: _data/pricing.yml [hunk 2]
```diff
@@ -9,3 +9,9 @@ plans:
...
+      - { text: "Weekly office hours to answer questions and best practices." }
```
Comment (davemcorwin) [inline _data/pricing.yml:15]: Wonder if this one should be separated into 2. The best practices one could be expanded to talk about pre-rendered or static site gen, jamstack, (not necessarily all the jargon) or generally the architecture supported by Federalist.

**Review by davemcorwin — COMMENTED** [review]
Can't comment on the pricing, but left some thoughts.

File: _data/pricing.yml (hunk 2 above)
Comment (apburnes) [reply _data/pricing.yml:15]: I like that. Let me rework it and see what we can get.

File: _data/pricing.yml [hunk 3]
```diff
@@ -9,3 +9,10 @@ plans:
       - { text: Automatic HTTPS certificates and renewals for all your websites. }
       - { text: Unlimited sites for your office. }
       - { text: No charges for storage or bandwidth. }
+  - title: "Platform Support $27,500/100 hours"
+    features:
+      - { text: "Personalized on-boarding with review of Federalist features and Github integration." }
+      - { text: "Weekly office hours to answer platform questions." }
+      - { text: "Implement best practices for static site generation, git-based CMS architecture, and Federalist deployments." }
```
Comment (davemcorwin) [inline _data/pricing.yml:16]:
```suggestion
      - { text: "Hands-on guidance to help implement best practices for static site generation, git-based CMS architecture, and Federalist deployments." }
```

**Review by davemcorwin — COMMENTED** [review]
If this is accurate and ok with Ron and #cg-business it's ok with me.

**Comment by davemcorwin** [comment]
If we are not ready to publish these changes, can we close this PR? @apburnes @RonWilliams

**Comment by apburnes** [comment]
We'll revisit this at a later date.

**Pull request closed without merging**